project/api/models.py

```diff
@@ -33,8 +33,16 @@ class TerritorialEntity(models.Model):
     """
 
     wikidata_id = models.PositiveIntegerField()  # Excluding the Q
+    label = models.TextField(max_length=90)
     color = ColorField()
     admin_level = models.PositiveIntegerField()
+    inception_date = models.DecimalField(decimal_places=1, max_digits=10)
+    dissolution_date = models.DecimalField(
+        decimal_places=1, max_digits=10, blank=True, null=True
```

Comment:
I was thinking thinking the dates as a user input but of course since we have gaps in wikidata both date fields should have less restriction. I guess they should be nullable? 

I need to convert the dates using the library in the viewset I think. Will take care of it.

What should references be changed as. Citations comes to mind.